x/inflation/types/params.go

```diff
@@ -7,20 +7,22 @@ import (
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	"gopkg.in/yaml.v2"
+
 	epochtypes "github.com/tharsis/evmos/x/epochs/types"
-	yaml "gopkg.in/yaml.v2"
 )
 
 // Parameter store keys
 var (
-	KeyMintDenom                            = []byte("MintDenom")
-	KeyGenesisEpochProvisions               = []byte("GenesisEpochProvisions")
-	KeyEpochIdentifier                      = []byte("EpochIdentifier")
-	KeyReductionPeriodInEpochs              = []byte("ReductionPeriodInEpochs")
-	KeyReductionFactor                      = []byte("ReductionFactor")
-	KeyPoolAllocationRatio                  = []byte("PoolAllocationRatio")
-	KeyDeveloperRewardsReceiver             = []byte("DeveloperRewardsReceiver")
-	KeyMintingRewardsDistributionStartEpoch = []byte("MintingRewardsDistributionStartEpoch")
+	KeyMintDenom                          = []byte("MintDenom")
+	KeyGenesisEpochProvisions             = []byte("GenesisEpochProvisions")
+	KeyEpochIdentifier                    = []byte("EpochIdentifier")
+	KeyReductionPeriodInEpochs            = []byte("ReductionPeriodInEpochs")
+	KeyReductionFactor                    = []byte("ReductionFactor")
+	KeyPoolAllocationRatio                = []byte("PoolAllocationRatio")
+	KeyTeamVestingProvision               = []byte("TeamVestingProvision")
+	KeyTeamVestingReceiver                = []byte("TeamVestingReceiver")
+	KeyMintingRewardsAllocationStartEpoch = []byte("MintingRewardsAllocationStartEpoch")
```

Comment:
let's use `iota` instead for the prefixes

Reply:
@fedekunze do you mean like this?
```
// Parameter store keys
const (
	KeyMintDenom = iota + 1
	KeyGenesisEpochProvisions
	KeyEpochIdentifier
	KeyReductionPeriodInEpochs
	KeyReductionFactor
	KeyPoolAllocationRatio
	KeyTeamVestingProvision
	KeyTeamAddress
	KeyMintingRewardsAllocationStartEpoch
)
```

How do I deal with the ParamSetPairs method then? Can I pass bytes like this? 
```
// Implements params.ParamSet
func (p *Params) ParamSetPairs() paramtypes.ParamSetPairs {
	return paramtypes.ParamSetPairs{
		paramtypes.NewParamSetPair([]byte{KeyMintDenom}, &p.MintDenom, validateMintDenom),
		paramtypes.NewParamSetPair([]byte{KeyGenesisEpochProvisions}, &p.GenesisEpochProvisions, validateGenesisEpochProvisions),
		paramtypes.NewParamSetPair([]byte{KeyEpochIdentifier}, &p.EpochIdentifier, epochtypes.ValidateEpochIdentifierInterface),
		paramtypes.NewParamSetPair([]byte{KeyReductionPeriodInEpochs}, &p.ReductionPeriodInEpochs, validateReductionPeriodInEpochs),
		paramtypes.NewParamSetPair([]byte{KeyReductionFactor}, &p.ReductionFactor, validateReductionFactor),
		paramtypes.NewParamSetPair([]byte{KeyPoolAllocationRatio}, &p.InflationDistribution, validateInflationDistribution),
		paramtypes.NewParamSetPair([]byte{KeyTeamVestingProvision}, &p.TeamVestingProvision, validateTeamVestingProvision),
		paramtypes.NewParamSetPair([]byte{KeyTeamAddress}, &p.TeamAddress, validateTeamAddress),
		paramtypes.NewParamSetPair([]byte{KeyMintingRewardsAllocationStartEpoch}, &p.MintingRewardsAllocationStartEpoch, validateMintingRewardsAllocationStartEpoch),
	}
}

```

Reply:
yeah, check the erc20 module for reference